tests/homework5/resources/full_treap.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
For me, it looks strange that you have separated resources folder for different tests. I mean you can create one resource folder and create subfolders in this folder for different tests. What do you think?